index.html

```diff
@@ -7,104 +7,175 @@
+            <p>I can help you build a product , feature or website. Look <br> through some of my work and experience! If
+                you
+                like what <br> you see and have a project you need coded, <br>
+                don’t hesitate to contact me.</p>
+        </div>
+        <div class="corner-img"></div>
+        <!-- <img class="corner-img" src="images/Illustration-header.png" alt=""> -->
     </section>
-    <section class="project-container">
-        <div class="project">
-            <h2>Profesional Art Printing Data</h2>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
-                industry's standard</p>
-            <ul>
-                <li>html</li>
-                <li>bootstrap</li>
-                <li>Ruby</li>
-            </ul>
-            <button>See Project</button>
+    <section>
+
+        <div class="my-recent-work">
+            <div style="flex-grow: 1;"></div>
+            <div class="recent-work-content">
+                <h2>My Recent Work</h2>
+                <hr>
+            </div>
+            <div style="flex-grow: 1;"></div>
         </div>
+        <section class="multi-post-stories">
+            <img src="images/Img Placeholder.png" alt="">
+            <div>
+                <h1>Multi-Post Stories</h1>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry's
+                    standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="container-list">
+                    <li>css</li>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button class="see-project-btn">See Project</button>
+            </div>
+        </section>
     </section>
     <section class="project-container">
-        <div class="project">
-            <h2>Profesional Art Printing Data</h2>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
-                industry's standard</p>
-            <ul>
-                <li>html</li>
-                <li>bootstrap</li>
-                <li>Ruby</li>
-            </ul>
-            <button>See Project</button>
+        <div class="project card2">
+            <div class="card-content">
+                <h2>Data Dashboard Healthcare</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry's standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            </div>
+            <div id="btn">
+                <button>See Project</button>
+            </div>
+
```

Comment:
- [ ] [OPTIONAL] You have showed great effort by implementing the responsiveness of the cards 👍  👏  👏 . However, there are a few things to apply to make it responsive in all the desktop sizes. Kindly ensure that the `width` of `card2` is not fixed to enable the card adapt all the sizes 👌 . 
- If you notice the languages are also not responsive. Please ensure that they also don't go beyond the cards. This will help to prevent the application from breaking at different breakpoints. Kindly refer to the image attached.

Your design 😎 
![languages cards](https://user-images.githubusercontent.com/99394374/213858726-d4c41288-19f2-42ad-88cb-fa903b274e89.PNG)